concrete/authentication/facebook/form.php

```diff
@@ -16,19 +16,43 @@
 
 if ($user->isLoggedIn()) {
     ?>
-    <div class="form-group">
+
+    <?php
+    $authenticationType = \Concrete\Core\Authentication\AuthenticationType::getByHandle('facebook');
```

Comment:
This needs to be passed in by the controller. Just add a `form` method and do `$this->set('authenticationType', $this->getAuthenticationType())`

Reply:
https://github.com/concrete5/concrete5/pull/6018/commits/70b4e410f0c048d82fb1460135312952a181f1f2

Reply:
:+1: